grunt_tasks/webpack.config.js

```diff
@@ -72,52 +72,78 @@ module.exports = {
         new webpack.NoEmitOnErrorsPlugin()
     ],
     module: {
-        loaders: [
+        rules: [
             // ES2015
             {
-                test: /\.js$/,
-                include: loaderPaths,
-                loader: 'babel-loader',
-                exclude: /node_modules/,
-                query: {
-                    presets: [es2015Preset],
-                    env: {
-                        cover: _coverageConfig()
+                resource: {
+                    test: /\.js$/,
+                    include: loaderPaths,
+                    exclude: /node_modules/
+                },
+                use: [
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: [es2015Preset],
+                            env: {
+                                cover: _coverageConfig()
+                            }
+                        }
                     }
-                }
+                ]
             },
             // JSON files
             {
-                test: /\.json$/,
-                include: loaderPaths.concat(loaderPathsNodeModules),
-                loader: 'json-loader'
+                resource: {
+                    test: /\.json$/,
+                    include: loaderPaths.concat(loaderPathsNodeModules)
+                },
+                use: [
+                    'json-loader'
+                ]
```

Comment:
According to https://webpack.js.org/guides/migrating/#json-loader-is-not-required-anymore this shouldn't be necessary any more.